libvast/vast/hash/uniquely_represented.hpp

```diff
@@ -38,10 +39,10 @@ struct is_uniquely_represented<std::tuple<Ts...>>
                        && ((0 + ... + sizeof(Ts)) == sizeof(std::tuple<Ts...>))> {
 };
 
-template <class T, size_t N>
+template <class T, std::size_t N>
```

Comment:
Why is that needed here?

Reply:
I think something in `fwd.hpp` used to pull `size_t` into the global namespace (e.g. the `<algorithm>` header has that effect), so without it it leads to compile errors from some translation units here. The alternative would be including `fwd.hpp`, but this seems more minimal.

Reply:
`size_t` is defined in `<cstddef>`. It's guaranteed to be the same type as `std::size_t`.